src/BlockTemplatesController.php

```diff
@@ -4,6 +4,7 @@
 use Automattic\WooCommerce\Blocks\Domain\Package;
 use Automattic\WooCommerce\Blocks\Templates\BlockTemplatesCompatibility;
```

Comment:
This is unused in favor of `SingleProductTemplateCompatibility`
```suggestion
```

Reply:
Fixed via 10bc1b15c09a7d82e067c1c68befe0ecd57981fa